code/game/machinery/newscaster.dm

```diff
@@ -873,6 +878,34 @@ var/list/obj/machinery/newscaster/allCasters = list() //Global list that will co
 		qdel(P)
 
 
+/obj/machinery/newscaster/proc/parsepencode(t, mob/user)
```

Comment:
Read what I [discussed with RemieRichards](https://github.com/tgstation/-tg-station/pull/7287#issuecomment-71113858).
TL;DR: I'll fix it once they are merged.